quadro_solve.cpp

```diff
@@ -0,0 +1,97 @@
+/**
+ * @brief This file contains functions that calculate the number of roots of a quadratic equation
+ * and their values, if any exist
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+#include <assert.h>
+double DBL_EPSILON = 1e-6;
+
+enum NumOfSolutions {
+    INF_SOLUTIONS = -1,
+    ZERO_SOLUTIONS = 0,
+    ONE_SOLUTION = 1,
+    TWO_SOLUTIONS = 2
+};
+
+enum TrueFalse {
+    TRUE = 1,
+    FALSE = 0
+};
+
+/**
+ * @brief A function that compares double with zero
+ * @param term
+ * @return TrueFalse
+ */
+
+unsigned short zeroComprasion(double term) {
```

Comment:
Just write a comparator of double values, i.e. a function that takes two double values and returns 0 if they are equal, negative integer if number1 < number2, and positive if number1 > number2.